pypeit/wavetilts.py

```diff
@@ -60,10 +68,16 @@ class WaveTilts(datamodel.DataContainer):
                                     descr='Order for spectral fit (nslit)'),
                  'func2d': dict(otype=str, descr='Function used for the 2D fit'),
                  'PYP_SPEC': dict(otype=str, descr='PypeIt spectrograph name'),
-                 'spat_flexure': dict(otype=float, descr='Flexure shift from the input TiltImage')}
+                 'slits_calib_key': dict(otype=str, descr='SlitTraceSet calibration key. '
```

Comment:
Can we just add the necessary bits to the data model to avoid having to read that Slits from disk?

Reply:
As I said in my other comment, I prefer not to add the slits quantities to the data model. If we read Slits from disk we can plot the tweaked slit edges, otherwise we would be able to save to the data model only the initial slit edges.